### ISSUE TITLE
feature req : input prompt in any location

### ISSUE THREAD

**Issue by b-y-f** [npiv/chatblade#34]
like chat -p mypropmt.txt -s -c 4 hi

**Comment by b-y-f** [comment]
currently
```
Prompt prompt.txt not found in /home/xx/.config/chatblade/prompt.txt

```

**Comment by b-y-f** [comment]
how want to save prompt in .config/xx/xx

**Comment by npiv** [comment]
Saving them under .config is to not have the type the full path each time, and additionally to not have them scattered across various locations. I use chatblade in various different projects, and have common prompts for them.

But I think its a fair use case to be to be able to just point to a local file as well. Will add that in

**Comment by npiv** [comment]
functionality available in latest v0.2.0
